### PR TITLE
Document clear-cache cookie in versions 9 and 10

### PR DESCRIPTION
**Pull request by EnasAbdelrazek**

### CODE REVIEW

**Review by MarkvanMents — APPROVED** [review]
Looks good to me.
Approving.
Since this is for two versions, I will publish it with 10.13 as being the latest version.

File: content/en/docs/refguide/runtime/mendix-client/_index.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ When the Mendix client is running, it sets a number of technical cookies to reco
 | **XASSESSIONID**<sup><small>2</small></sup> | Runtime | Holds the ID of the user's session | `/` | 1 year for offline sessions<sup><small>1</small></sup>, otherwise based on the session timeout |  `true` | 
 | **useAuthToken** | Runtime | Lets the runtime know whether to generate an authtoken or not—*introduced for apps created in Mendix version 10.9.0 and above* | `/` | until the end-user closes their browser (session cookie) |  `true` | 
 | **authtoken** | Runtime | Used to hold the authentication token—*introduced for apps created in Mendix version 10.9.0 and above* | `/` | until the authentication token expires |  `true` | 
+| **clear_cache** | Runtime | Instructs the client to clear the cached session data whenever a new end-user signs in to an offline application—*introduced for apps created in Mendix version 10.13.0 and above* | `/` | until client starts or end-user closes their browser | `false` |
```
Comment (Robinfr) [inline content/en/docs/refguide/runtime/mendix-client/_index.md:266]: @EnasAbdelrazek This is not offline specific?

File: content/en/docs/refguide9/runtime/mendix-client.md [hunk 2]
```diff
@@ -228,6 +228,7 @@ When the Mendix client is running, it sets a number of technical cookies to reco
 | **xasid** | Runtime | Used for multi-instance fail-over | `/` | until user closes their browser (session cookie) |  `true` | 
 | **reloginReason** | Runtime | Used to let the client know that a relogin reason should be displayed on the sign in page | `/` | until user closes their browser (session cookie) |  `true` | 
 | **XASSESSIONID**<sup><small>2</small></sup> | Runtime | Holds the ID of the user's session | `/` | 1 year for offline sessions<sup><small>1</small></sup>, otherwise based on the session timeout |  `true` | 
+| **clear_cache** | Runtime | Instructs the client to clear the cached session data whenever a new end-user signs in to an offline application—*introduced for apps created in Mendix version 9.24.24 and above* | `/` | until client starts or end-user closes their browser | `false` |
```
Comment (Robinfr) [inline content/en/docs/refguide9/runtime/mendix-client.md:231]: Same comment

**Review by Robinfr — CHANGES_REQUESTED** [review]

**Comment by Robinfr** [comment]
@EnasAbdelrazek Should we mention the better approach for 10? Ideally, people shouldn't use the cookie directly. Maybe we should reference the new API in the description of the cookie for Mx10?

File: content/en/docs/refguide/runtime/mendix-client/_index.md [hunk 3]
```diff
@@ -263,7 +263,7 @@ When the Mendix client is running, it sets a number of technical cookies to reco
...
 | **authtoken** | Runtime | Used to hold the authentication token—*introduced for apps created in Mendix version 10.9.0 and above* | `/` | until the authentication token expires |  `true` | 
-| **clear_cache** | Runtime | Instructs the client to clear the cached session data whenever a new end-user signs in to an offline application—*introduced for apps created in Mendix version 10.13.0 and above* | `/` | until client starts or end-user closes their browser | `false` | 
+| **clear_cache** | Runtime | Instructs the client to clear the cached session data whenever a new end-user signs in to an offline application. If the client is not directly used for the sign-in functionality, you can use the newly added Java API, [addSessionCookies](https://apidocs.rnd.mendix.com/10/runtime/com/mendix/m2ee/api/IMxRuntimeResponse.html#addSessionCookies), which sets that cookie a long with all the necessry session cookies.—*introduced for apps created in Mendix version 10.13.0 and above* | `/` | until client starts or end-user closes their browser | `false` |
```
Comment (Robinfr) [inline content/en/docs/refguide/runtime/mendix-client/_index.md]: >which sets that cookie a long with all the necessry session cookies

should be 
> which sets all necessary cookies related to the session.

**Review by Robinfr — CHANGES_REQUESTED** [review]

**Review by Robinfr — APPROVED** [review]

**Comment by EnasAbdelrazek** [comment]
We have to update the Java API reference after releasing 10.13

**Pull request merged**